### PR TITLE
refactor: replace insertSemicolonCommand with additionalTextEdits

### PR DESCRIPTION
**Pull request by antico5**
As apart of the discarded work for command server execution, there was this refactor which is nice to have. Now, instead of relying on a command for testing and appending semicolon at the end of import lines, we make use of the `additionalTextEdits` field of completion items.

### CODE REVIEW

File: server/src/services/completion/getImportPathCompletion.ts [hunk 1]
```diff
@@ -50,13 +57,32 @@ export function getImportPathCompletion(
...
+      line: position.line,
+      character: 0,
+    },
+    end: {
+      line: position.line,
+      character: 99999,
+    },
+  });
+
+  if (!/;\r?\n?$/.test(line ?? "")) {
```
Comment (kanej) [inline server/src/services/completion/getImportPathCompletion.ts:74]: Is this case covered by any tests?

Comment (antico5) [reply server/src/services/completion/getImportPathCompletion.ts:74]: I  think it's not. I'll add one now.

Comment (antico5) [reply server/src/services/completion/getImportPathCompletion.ts:74]: @kanej done!

**Review by kanej — APPROVED** [review]
LGTM!

**Pull request merged**